tests/src/test/scala/whisk/core/controller/test/ActionsApiTests.scala

```diff
@@ -1043,13 +1043,15 @@ class ActionsApiTests extends ControllerTestCommon with WhiskActionsApi {
       status should be(Accepted)
       val response = responseAs[JsObject]
       response.fields("activationId") should not be None
+      headers.exists(_.is(ActivationIdHeaderInLowercase)) should be(true)
```

Comment:
headers are immutable and per the line above you know the correct activation id. You should be able to rewrite this to something like:

```scala
headers should contain(RawHeader(ActivationIdHeader, response.fields("activationId").convertTo[String]))
```

That'll give you a more explanatory response in tests. WDYT?